src/pages/index.js

```diff
@@ -13,6 +14,7 @@ export default function Home() {
       <main className={styles.main}>
         <div className={styles.description}>
           <div className={styles.title}>HELLO</div>
+          <LogInModal></LogInModal>
```

Comment:
Why is the login modal here, when you already have it in _app.js?

Reply:
That's a good question, I don't think it does anything, not really sure why it ended up there. Will remove it.